woocommerce/woocommerce
# Fix: use correct wrapper class for product grids

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




This PR fixes the wrong class name in product grids blocks to fixes the styling issue with theme.json and global style. The affected blocks are:

- `woocommerce/handpicked-products`.
- `woocommerce/product-new`.
- `woocommerce/product-on-sale`.
- `woocommerce/product-by-attribute`.
- `woocommerce/product-top-rated`.
- `woocommerce/product-tag`.
- `woocommerce/product-category`.
- `woocommerce/product-best-sellers`.

Closes #48298.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Setup WordPress and WooCommerce
2. Add the "Newest products" block somewhere on the website
3. Edit the `theme.json` of the current theme, adding this under `"styles" -> "blocks"`:
```json
"woocommerce/product-new": {
    "elements": {
        "button": {
            "color": {
                "background": "red !important"
            }
        }
    }
}
```
4. Notice how the buttons are red both in the block editor and on the frontend.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
